pkgs/by-name/ez/ezstream/package.nix

```diff
@@ -7,28 +7,37 @@
   taglib,
   libxml2,
   pkg-config,
+  check,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "ezstream";
-  version = "0.6.0";
+  version = "1.0.2";
 
   src = fetchurl {
-    url = "https://ftp.osuosl.org/pub/xiph/releases/ezstream/${pname}-${version}.tar.gz";
-    sha256 = "f86eb8163b470c3acbc182b42406f08313f85187bd9017afb8b79b02f03635c9";
+    url = "https://ftp.osuosl.org/pub/xiph/releases/ezstream/ezstream-${finalAttrs.version}.tar.gz";
+    hash = "sha256-Ed6Jf0ValbpYVGvc1AqV072mmGbsX3h5qDsCQSbFTCo=";
   };
 
+  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    sed -i 's/#include <sys\/stat.h>/#include <stddef.h>\n&/' src/playlist.c
```

Comment:
Why not `substituteInPlace` here too ?

I thought there was an issue on darwin with `sed -i`, and had to use eg. `sed -i.bak`, but maybe this is not a problem with nix.